proxy.js

```diff
@@ -578,7 +579,13 @@ function balanceWorkers(){
     }
 }
 
-function enumerateWorkerStats(){
+function enumerateWorkerStats() {
+	// here we do a bit of a hack and "cache" the activeWorkers
```

Comment:
Should check for global.config.httpEnable before dumping to disk.

Reply:
I don't know where else in the app loop "activeWorkers" is properly defined, and I haven't tried too hard to figure it out, hence the "hack."